src/common/config.h

```diff
@@ -306,6 +306,7 @@ struct md_config_t {
   unsigned get_osd_pool_default_min_size() const {
     auto min_size = get_val<uint64_t>("osd_pool_default_min_size");
     auto size = get_val<uint64_t>("osd_pool_default_size");
+    assert(size > 1 && "osd_pool_default_size must be larger than 1");
```

Comment:
i think the question is: shall we have a lower bound of `2`? i'd suggest use `1` instead. please search "osd pool default size = 1" over the internet, people are using this setting for variant purposes.

probably we can avoid enforcing this policy in the form of `assert()` if possible.

so, i'd suggest use `.set_min(1)` in `options.cc`.